api/src/test/java/uk/gov/dhsc/htbhf/claimant/testsupport/VerificationResultTestDataFactory.java

```diff
@@ -71,6 +71,14 @@ public static VerificationResult aFullAddressMismatchVerificationResult() {
                 .build();
     }
 
+    public static VerificationResult anIneligibleFullAddressMismatchVerificationResult() {
```

Comment:
I disagree with this VerificationResult. As I said above, this is an invalid state to be in, if the EligibilityOutcome is NOT_CONFIRMED then all VerificationOutcomes will be NOT_SET.